src/main/webapp/app/localvc/repository-view/repository-view.component.ts

```
@@ -190,4 +193,27 @@ export class RepositoryViewComponent implements OnInit, OnDestroy {
     }
+
+    private loadAuxiliaryRepository(exerciseId: number, auxiliaryRepositoryId: number) {
+        this.programmingExerciseService
+            .findWithAuxiliaryRepository(exerciseId)
+            .pipe(
+                tap((exerciseResponse) => {
+                    this.exercise = exerciseResponse.body!;
+                    const auxiliaryRepo = this.exercise.auxiliaryRepositories?.find((repo) => repo.id === auxiliaryRepositoryId);
+                    if (auxiliaryRepo) {
+                        this.domainService.setDomain([DomainType.AUXILIARY_REPOSITORY, auxiliaryRepo]);
+                        this.repositoryUri = auxiliaryRepo.repositoryUri!;
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)